azext_iot/operations/hub.py

```diff
@@ -961,6 +961,8 @@ def _build_device_or_module_connection_string(device, key_type='primary', module
     raise CLIError('Unable to form target connection string')
 
 
+# Introducing breaking changes by removing this command as same command exist in core-cli.
+# Only removing the command and keeping command defination for further references.
```

Comment:
typo - should be "definition"

Reply:
Fixed in PR https://github.com/Azure/azure-iot-cli-extension/pull/66